packages/lesswrong/components/votes/VoteArrowIcon.tsx

```diff
@@ -91,7 +91,7 @@ const VoteArrowIcon = ({ solidArrow, strongVoteDelay, orientation, enabled = tru
   const Tooltip = enabled
     ? ({ children }) => children
     : ({ children }) => (
-      <LWTooltip title={"You do not have permission to vote on this"}>
+      <LWTooltip title={"You do not have permission to vote on this"} placement="top">
```

Comment:
This isn't related to the bug - I just think it looks a lot better when it's not covered by the mouse.